docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -123,65 +125,52 @@ image::connectors-diagram.png[Illustration of data flow from data source through
+* *Access rules* for the service account that defines how your {connectors} instances can access and use the topics in your Kafka instance.
 endif::[]
 
+ifdef::qs[]
 .Procedure
-. Create a Kafka topic for your connectors:
-.. In the OpenShift Application Services web console, select *Streams for Apache Kafka* > *Kafka Instances*.
-.. Click the name of the {product-kafka} instance that you created for connectors.
-.. Select the *Topics* tab, and then click *Create topic*.
-.. Type a unique name for your topic. For example, type *test-topic* for *Topic Name*.
-.. Accept the default settings for partitions, message retention, and replicas.
-. Create a service account for connectors:
-.. In the web console, select *Service Accounts*, and then click *Create service account*.
-.. Type a unique service account name (for example, *test-service-acct* ) and then click *Create*.
-.. Copy the generated *Client ID* and *Client Secret* to a secure location. You'll use these credentials to configure connections to this service account.
-.. Select the *I have copied the client ID and secret* option, and then click *Close*.
-
-. Set the level of access for your new service account in the Access Control List (ACL) of the Kafka instance:
-.. Select *Streams for Apache Kafka* > *Kafka Instances*.
-.. Click the name of the Kafka instance that you want the service account to access.
-.. Click the *Access* tab to view the current ACL for the Kafka instance and then click *Manage access*.
-.. From the *Account* drop-down menu, select the service account that you created in Step 2, and then click *Next*.
-.. Under *Assign Permissions*, click *Add permission*.
-.. From the drop-down menu, select *Consume from a topic*. Set all resource identifiers to `is` and all identifier values to `"*"`.
-.. From the *Add permission* drop-down menu, select *Produce to a topic*. Set all resource identifiers to `is` and all identifier values to `"*"`.
-+
-The `is "*"` settings enable connectors that are configured with the service account credentials to produce and consume messages in any topic in the Kafka instance.
+Make sure that you have set up the prerequisite components.
 
-ifdef::qs[]
 .Verification
-* Did you create a topic for connectors?
-* Did you create a service account and save the credentials to a secure location?
-* Did you set the *Consume from a topic* and *Produce to a topic* permissions for the service account?
+* Is the Kafka instance listed in the Kafka instances table and is it in the *Ready* state?
+* Did you verify that your service account was successfully created in the *Service Accounts* page?
+* Did you save your service account credentials to a secure location?
+* Are the permissions for your service account listed in the *Access* page of the Kafka instance?
+* Is the Kafka topic that you created for {product-connectors} listed in the topics table of the Kafka instance?
+
+endif::[]
+
+ifndef::qs[]
+.Verification
```

Comment:
Do we need verification for prereqs? It should be enough just to list them.

Reply:
Yes - this was specifically requested by the PM. We are trying it out for Service Preview.